.github/workflows/openvex.yaml

```diff
@@ -0,0 +1,27 @@
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - 'v*.*.*'
+
+
+jobs:
+  vexctl:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Set tag name
+        shell: bash
+        run: |
+          echo "TAG=${{ github.ref_name }}" >> "$GITHUB_ENV"
```

Comment:
This still allows script injection. Can we directly assign the `${{ github.ref_name }}` in the env block like

```yaml
jobs:
  vexctl:
    runs-on: ubuntu-latest
    env:
      TAG: ${{ github.ref_name }}
    steps:
    ...
```